tower/src/retry/backoff.rs

```diff
@@ -0,0 +1,228 @@
+//! This module contains generic backoff utlities to be used with the retry
+//! layer.
+//!
+//! The [`Backoff`] trait is a generic way to represent backoffs that can use
+//! any timer type.
+//!
+//! [`ExponentialBackoff`] implements [`Backoff`] and provides a batteries
+//! included exponential backoff and jitter strategy.
+
+use rand::Rng;
+use rand::{rngs::SmallRng, thread_rng, SeedableRng};
+use std::fmt::Display;
+use std::future::Future;
+use std::time::Duration;
+use tokio::time;
+
```

Comment:
the doc comment either for the module or the `Backoff` trait should _probably_ at least mention what the term "backoff" means, for readers who've never seen it before?